machine/wnc_x86_64/wnc.patch/build-config/make/images.make

```diff
@@ -0,0 +1,370 @@
+#-------------------------------------------------------------------------------
```

Comment:
What is this file for?  You should not have to patch build-config/make/images.make.

What is the default images.make lacking that you require?